### DISCUSSION TITLE
URL query ?

### DISCUSSION THREAD

**Discussion by francois-lecoquierre** [vidboda/MobiDetails#26] [Q&A] [answered]
Hello, 
As a novice user of MobiDetails, i wonder if queries can be submitted via the URL, something like:
https://mobidetails.iurc.montp.inserm.fr/MD/query=NM_006624.5:c.831+5G>A

That would allow an easy accessibility via VBA or other resources. 
Sorry if this info is easily available !
Thanks !
François

**Comment by vidboda** [comment, marked as answer]
Hi François,

queries can be submitted via URL using the API.
There is a swagger implementation of the API for testing purpose available at:

https://mobidetails.iurc.montp.inserm.fr/MDAPI/

However you will need an account to generate new annotations  via the API (API key required for several methods).

I hope this will suit your needs.
